static-responses/README.md

```diff
@@ -0,0 +1,64 @@
+# Static Responses
+
+There are situations where no backend requests are sent for an endpoint.
+Nevertheless, a response is needed. So a `response` block can exist in an
+`endpoint` block without any `proxy` or `request` blocks.
+
+Consider the following example, in which the client is instructed to perform a
+redirect via a `303` status code an a `location` header specifying the redirect
+target:
+
+```hcl
+    endpoint "/redirect" {
+      response {
+        status = 303
+        headers = {
+          location = "https://www.example.com/"
+        }
+      }
+    }
+```
+
+Call Couper:
+
+```shell
+$ curl -i http://localhost:8080/redirect
+HTTP/1.1 303 See Other
+Location: https://www.example.com/
+...
+
+```
+
```

Comment:
we could add a simple "configuration" endpoint that emits some env variables:

```hcl
  endpoint "/app/conf" {
    response {
      json_body = {
        version = env.APP_VERSION
        env = env.APP_ENV
        debug = env.APP_DEBUG == "true"
      }
    }
  }
```

many SPAs need a bit of info/settings for bootstrapping. this could be a simple pattern for this need.